robo.yml

```diff
@@ -37,6 +37,11 @@ data:
     file: "eurovoc_in_skos_core_concepts.rdf"
     format: "zip"
 
+  - name: "digital-europa-thesaurus"
```

Comment:
Shouldn't this be `europa-digital-thesaurus`?

Reply:
Yes it should be :)